Dockerfile

```diff
@@ -55,12 +61,11 @@ RUN \
   echo "**** setup wizard ****" && \
   mkdir -p /wizard && \
   if [ -z ${KASM_VERSION+x} ]; then \
-    KASM_VERSION=$(curl -sX GET 'https://api.github.com/repos/kasmtech/kasm-install-wizard/releases' \
-    | jq -r 'map(select(.prerelease)) | sort_by(.published_at) | last | .tag_name'); \
+      KASM_VERSION=$(curl -sX GET https://kasm-ci.s3.amazonaws.com/dev-version.txt); \
   fi && \
   curl -o \
     /tmp/wizard.tar.gz -L \
-    "https://github.com/kasmtech/kasm-install-wizard/archive/refs/tags/${KASM_VERSION}.tar.gz" && \
+    "https://github.com/kasmtech/kasm-install-wizard/tarball/feature/KASM-3512-gpu-option" && \
```

Comment:
Is this supposed to be a fixed string and not using KASM_VERSION?

Reply:
Yes for now this will be a fixed string on develop. Once works is completed on the upgrade methodology built into the node app this will get re pointed to the releases.